atlasdb-jepsen-tests/src/test/java/com/palantir/atlasdb/jepsen/utils/TestEventUtils.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.jepsen.utils;
+
+import com.palantir.atlasdb.jepsen.events.ImmutableFailEvent;
+import com.palantir.atlasdb.jepsen.events.ImmutableInfoEvent;
+import com.palantir.atlasdb.jepsen.events.ImmutableInvokeEvent;
+import com.palantir.atlasdb.jepsen.events.ImmutableOkEvent;
+import com.palantir.atlasdb.jepsen.events.OkEvent;
+import com.palantir.atlasdb.jepsen.events.RequestType;
+
+public abstract class TestEventUtils {
+
+    private static final String LOCKNAME = "default_lockname";
+    private static final String TIMESTAMP = "timestamp";
+
+    public static ImmutableInvokeEvent invokeLock(long time, int process, String lockname) {
```

Comment:
Generally we prefer methods to return the interfaces (so `InvokeEvent`, etc.) as opposed to (`ImmutableInvokeEvent` etc.). There isn't any specific reason why we want the events created to be from Immutables.